docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
       dockerfile: Dockerfile
     volumes:
       - ./:/chroma
-      - index_data:/index_data
+      # Be aware that indexed data are located in "/chroma/chroma/"
+      # Default configuration for persist_directory in chromadb/config.py
     command: uvicorn chromadb.app:app --reload --workers 1 --host 0.0.0.0 --port 8000 --log-config log_config.yml
     environment:
       - IS_PERSISTENT=TRUE
```

Comment:
I think we should go one step further and instead of simply removing the dir we should instead add `- PERSIST_DIRECTORY=${PERSIST_DIRECTORY:-/chroma/chroma}` while this is not a huge change it does give the indication to someone reading the docker compose where is actually data persisted without having to read through the code.

Reply:
great suggestion!

Reply:
@Benvii did this suggestion get rewinded?